lib/curl/README.md

```diff
@@ -0,0 +1,65 @@
+# C API client for skycoin_rest_api/
+
+## Overview
+This API client was generated by the [OpenAPI Generator](https://openapi-generator.tech) project. By using the [OpenAPI spec](https://openapis.org) from a remote server, you can easily generate an API client.
+
+- API version: 0.25.1
+- Package version: 
+- Build package: org.openapitools.codegen.languages.CLibcurlClientCodegen
+For more information, please visit [http://127.0.0.1:6420](http://127.0.0.1:6420)
+
+## Installation
+You'll need the `curl 7.58.0` package in order to build the API. To have code formatted nicely you also need to have uncrustify version 0.67.
+
+# Prerequisites
+
+## Install the `curl 7.58.0` package with the following command on Linux.
```

Comment:
@stdevMac [looking at the code](https://github.com/simelo/libskycoin/blob/6d6436361a7a7d30e89a8be5406ac64f3bdb7acb/lib/curl/CMakeLists.txt#L13) I get noticed that `cmake` it's looking for a library not a program an the version should be **at least** `7.58.0` so is not strict to this version(the `cmake` error msg is very specific "Could NOT find CURL ... Required is at least version "7.58.0"" )
So solving this dependency with the platform package manager could work in most cases, in any case if you need detailed instructions put a link to the package vendor, and please if you put instructions for specifics plattforms(`sudo apt remove curl`, what about redhat?) mentions install `x`. In the debian family you can run for example...

Reply:
@stdevMac the library could be `libcurl4-gnutls-dev` for example.